central/debug/service/service.go

```diff
@@ -432,6 +435,18 @@ type dbExtension struct {
 	ExtensionVersion string `json:"ExtensionVersion"`
 }
 
+// delegatedRegistryConfig represents delegated scanning configuration
+type delegatedRegistryConfig struct {
+	EnabledFor       string              `json:"enabled_for,omitempty"`
+	DefaultClusterID string              `json:"default_cluster_id,omitempty"`
+	Registries       []delegatedRegistry `json:"registries,omitempty"`
+}
+
+type delegatedRegistry struct {
+	Path      string `json:"path,omitempty"`
+	ClusterID string `json:"cluster_id,omitempty"`
+}
+
```

Comment:
Any reason why we wouldn't use the existing generated types? Be less error prone in the future if we ever make changes.

Reply:
I will give it a try